src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Takes in command from user and checks what to do with it
+ */
+public class Parser {
+    TaskList taskList = new TaskList();
+
+    Parser() {
+    }
+
+    /**
+     * Check if description is missing
+     *
+     * @param input
+     * @return validity
+     */
+    private boolean checkInvalidDescription(String input) {
+        if (input.trim().equals("todo") || input.trim().equals("deadline") || input.trim().equals("event")) {
```

Comment:
This is perfectly valid, but I think it would be better to trim the input once beforehand then compare to the three cases:

```

String trimmedInput = input.trim();
if (trimmedInput.equals("todo") || trimmedInput.equals("deadline") || trimmedInput.equals("event")) {
    //rest of the code

```